src/app/styles/chat.css

```
@@ -10,109 +10,168 @@
+   dedicated highlight token (pure black on light, pure white on dark) — a
+   wide text-3→text ramp reads as a faint tide, not motion. linear so the
+   sweep never stalls at the loop edges. */
+.chat-activity__label--shimmer {
+  background: linear-gradient(90deg, var(--text-3) 40%, var(--shimmer-hi) 50%, var(--text-3) 60%);
+  background-size: 300% 100%;
+  -webkit-background-clip: text;
+  background-clip: text;
+  color: transparent;
+  animation: chatShimmer var(--dur-loop) linear infinite;
+}
+@keyframes chatShimmer {
```

Stylelint (17.14.0)

[error] Expected keyframe name "chatShimmer" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)